### PR TITLE
Update 02-common-mistakes.md

### PR DESCRIPTION
**Pull request by laurasootes**
To solve #160  I replaced the html table by a common markdown table, while keeping the contents the same.
This fixes the problem that the table was rendered within the next header line

### CODE REVIEW

File: episodes/02-common-mistakes.md [hunk 1]
```diff
@@ -162,38 +162,13 @@ that are excessively long. Including the units in the field names avoids confusi
...
-	<td> 1st Inf</td>
-</tr>
-<tr>
-	<td> age_18 </td>
-	<td> years18</td>
-	<td> 18years</td>
-</tr>
-</table>
+| Null Values  | Good alternative  |Avoid|  
+| -----------  | ------------------| -------------- |
```
Comment (bencomp) [inline episodes/02-common-mistakes.md:166]: I don't know for sure if the missing hyphens would have a big impact, but the first column name was copied incorrectly.

```suggestion
| Good Name    | Good Alternative  | Avoid          |
|--------------|-------------------|----------------|
```

**Review by bencomp — CHANGES_REQUESTED** [review]
Thanks, @laurasootes! I have just a few small suggestions for the header row.

File: episodes/02-common-mistakes.md (hunk 1 above)
Comment (laurasootes) [reply episodes/02-common-mistakes.md]: Hi @bencomp, thank for noticing the incorrect column names. I implemented you suggestion

**Review by bencomp — APPROVED** [review]
Of course my review is biased, but I'm happy anyway. Hopefully the maintainers are also happy with these changes.

**Comment by josenino95** [comment]
Thank you @bencomp and @laurasootes for taking care of this a while ago. I just merged PRs #167 and #171

**Comment by bencomp** [comment]
Thanks, @josenino95. Could you close #160 too, now that this fixed it?

**Pull request merged**